lib/Dialect/SV/Transforms/SVExtractTestCode.cpp

```diff
@@ -392,6 +392,25 @@ static bool isBound(hw::HWModuleLike op, hw::InstanceGraph &instanceGraph) {
   });
 }
 
+// Add any existing bindings to the bind table.
+static void addBinds(hw::HWModuleLike op, hw::InstanceGraph &instanceGraph,
+                     SmallVectorImpl<BindOp> &existingBinds,
+                     BindTable &bindTable) {
```

Comment:
For a future cleanup: consider refactoring this module to make these static methods private member functions of the pass to avoid the pass-by-reference of many things which the pass knows about already.

Reply:
Will send a refactor later on, there are a few things I'd like to clean up about this pass now that the new features are taking shape.